test/unauthorized.coffee

```diff
@@ -0,0 +1,135 @@
+expect = require('chai').expect
+
+command = require './helpers/command'
+TestApi = require './helpers/test_api'
+assertStdout = require './helpers/assert_stdout'
+
+gracefulUnauthorized =
+  """
+  You need to log in for that.
+  Type closeheat login or open http://app.closeheat.com/api/login to do it swiftly.
```

Comment:
http://app.closeheat.com/api/login  ? :D I think you should leave only : `closeheat login`